catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -490,24 +493,39 @@ private static void setMetacardEmailAttribute(List<EmailAddressType> emailAddres
 
     private static void setMetacardPhoneNumberAttribute(List<TelephoneNumberType> phoneNumbers,
             String metacardAttribute, MetacardImpl metacard) {
+
         List<String> metacardPhoneNumbers = new ArrayList<>();
+        if (CollectionUtils.isNotEmpty(phoneNumbers)) {
+            for (TelephoneNumberType digits : phoneNumbers) {
+                metacardPhoneNumbers.add(getPhoneNumber(digits));
 
-        for (TelephoneNumberType digits : phoneNumbers) {
+                if (CollectionUtils.isNotEmpty(metacardPhoneNumbers)) {
+                    metacard.setAttribute(metacardAttribute, (Serializable) metacardPhoneNumbers);
+                }
+            }
+        }
+    }
 
-            if (StringUtils.isBlank(digits.getExtension())) {
-                metacardPhoneNumbers.add(String.format("(%s) %s",
-                        digits.getAreaCode(),
-                        digits.getNumber()));
+    private static StringBuilder buildNonNullString(StringBuilder stringBuilder,
+            String attributeValue, String format, String delimiterBefore) {
```

Comment:
Oh gotcha, yeah, now that I look at it this is pretty strange. I'll fix this up. 
As for your other comment, could you expand on that some more as well?